app/controllers/organen/index.js

```diff
@@ -1,22 +1,61 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
-import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
 
-export default class MandatenbeheerBestuursorganenNewController extends Controller {
-  @service router;
+export default class OrganenIndexController extends Controller {
+  queryParams = [
+    'sort',
+    'activeFilter',
+    'selectedTypes',
+    'startDate',
+    'endDate',
+  ];
   @service store;
   @service decretaleOrganen;
 
+  @tracked sort = 'naam';
+  @tracked activeFilter = false;
+  @tracked selectedTypes = ['isDecretaal', 'notDecretaal'];
+  @tracked startDate;
+  @tracked endDate;
+
+  @tracked modal = false;
```

Comment:
Would name it `isModalActive` or something starting with is to adress that it is a boolean